twitter_bot/Commands/stats.py

```diff
@@ -6,11 +6,15 @@ def stats(text):
     r = req.get('https://www.osrsbotdetector.com/dev/site/dashboard/projectstats').text
     r = json.loads(r)
 
+    project_start_timestamp = 1614470400
     accounts = r['total_accounts']
     bans = r['total_bans']
     faccounts = "{:,}".format(accounts)
     fbans = "{:,}".format(bans)
-    days_since_start = int((time.time() - 1614470400)/(60*60*24))
+    days_since_start = int((time.time() - project_start_timestamp)/(60*60*24))
+    
+    if accounts == 0:
+        return 
```

Comment:
we need to return a response pretty sure

Reply:
resolved with commit 30797c815611cbc3699e32e2cf032d6198594952

Reply:
Edit: Response is validated, if Response is None nothing will be send
https://github.com/Bot-detector/Bot-detector-Twitter/blob/d5f10ed98e1b67718eacc9d7faec2dc9673d4438/twitter_bot/Tweet/Tweet.py#L62